x-pack/test/functional/apps/infra/helpers.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { apm, timerange } from '@kbn/apm-synthtrace-client';
+
+const SERVICE_PREFIX = 'service';
+// generates traces, metrics for services
+export function generateAddServicesToExistingHost({
```

Comment:
this depends on receiving a hardcoded host that i got from other test data.  would like to refactor to use synthtrace infra hosts with consistent host names.